sqreen/README.md

```diff
@@ -0,0 +1,48 @@
+# Sqreen Integration
+
+## Overview
+
+The Sqreen integration enables you and your team to monitor your service security activity right from Datadog.
+
+It provides you with dashboards, security rules, log facets and saved views.
+
+This integration is only available to Datadog AppSec design partners. 
```

Comment:
Is there somewhere to link that describes what an AppSec design partner is?

Reply:
@apigirl this integration will be fully private, meaning no accounts besides the design partners one should see it on the docs nor on the Dashboard. Knowing this, I'm not sure it's worth describing it. Wdyt?

Reply:
let's just say that it's only available to Datadog and Sqreen customers?

Reply:
Fixed